src/main/java/com/bringframework/exceptions/NoUniqueBeanException.java

```diff
@@ -0,0 +1,8 @@
+package com.bringframework.exceptions;
+
+/**
+ * Exception thrown when multiple beans of the same type are found but expected one.
```

Comment:
probaby `of the same type and name` would be more precise ?

Reply:
Not agree with this addition. Because we will never get this exception when we try to find bean by name.
All beans will be stored in Map structure where key is bean name and which means that one bean name will be a maximum one bean object and not more.

Reply:
Yes, good point, I agree!